repos/spack_repo/builtin/packages/octave/package.py

```diff
@@ -389,3 +394,9 @@ def setup_dependent_package(self, module, dependent_spec):
         """
         # Octave extension builds can have a global Octave executable function
         module.octave = Executable(join_path(self.spec.prefix.bin, "octave"))
+
+    def setup_run_environment(self, env):
+        if self.spec.satisfies("@10.2.0:"):
+            env.set("CXX", self.compiler.cxx + " -std=gnu++17")
```

Comment:
Not sure if this is a good default in general. See above.

Reply:
> Not sure if this is a good default in general. See above.

Well, without that pkg install -forge package fails.
Octave compilation is fine without setting -std=gnu++17, but the extended packages fails without that.

Reply:
Like I wrote above, that is because of the old Clang that you are using.